src/main/scala/it/pagopa/interop/authorizationprocess/common/Adapters.scala

```diff
@@ -247,4 +260,34 @@ object Adapters {
   implicit class JwkOtherPrimeInfoWrapper(private val o: JwkOtherPrimeInfo) extends AnyVal {
     def toApi: OtherPrimeInfo = OtherPrimeInfo(r = o.r, d = o.d, t = o.t)
   }
+
+  implicit class UserResourceWrapper(private val ur: SelfcareV2Dependency.UserResource) extends AnyVal {
```

Comment:
It can be deleted if we avoid to return the user model

Reply:
I need these to resolve Option None field

Reply:
now it can be deleted